client/components/additionalRepos/additional-repos.html

```diff
@@ -0,0 +1,22 @@
+<label>Additional repositories</label>
+<button class="btn btn-xs btn-success" type="button" ng-click="addRepository()">+</button>
```

Comment:
Lets use one of these icons instead of the +?
http://fontawesome.io/icon/plus/
http://fontawesome.io/icon/plus-circle/
http://fontawesome.io/icon/plus-square/